### DISCUSSION TITLE
[THEME] KDE Breeze

### DISCUSSION THREAD

**Discussion by eliandoran** [TriliumNext/Trilium#4400] [Show and tell]
Hi,

I've been working on a theme for quite some time and I think it's in a state that it can be presented.

Basically the theme aims to integrate Trilium Notes to KDE's default theme as much as possible.

Link: https://github.com/eliandoran/breeze-trilium

## Highlights

- Automatic light/dark mode
- Toolbar at the top instead of the left, more in line with the KDE design.
- Similarly, the ribbon has been moved from the top to the bottom with a drawer design.

## Screenshots

![Light theme](https://github.com/eliandoran/breeze-trilium/raw/master/etc/screenshots/breeze-light.png)

![Dark theme](https://github.com/eliandoran/breeze-trilium/raw/master/etc/screenshots/breeze-dark.png)

Let me know your thoughts. :)

**Comment by zadam** [comment]
Wow, that's hell of a theme! Extremely well done!

It occurred to me that Trilium (Box) icons are kind of similar to the KDE/Breeze, they don't look out of place.

The only thing I'm feeling kind of uneasy is that this had to require a lot of deep CSS customizations to move things around.

**Comment by sottey** [comment]
I have a lot of themes, and I customize the heck out of them. This is easily my favorite, and with no changes. Nicely done!!!!

**Comment by hypulse** [comment]
This theme is really the best. If it wasn't for this theme, I would have hesitated to use Trilium. I really want to leave a word of thanks!!!
At the same time, I don't think mobile support is necessary, but there is an issue where it's difficult to restore the theme as notes are not visible when entering the mobile version.
It seems fine to add such style as default:
```css
.mobile #root-widget {
  flex-direction: column !important;
}
```

**Reply by eliandoran** [reply]
@hypulse , really glad to hear that! :)

I've created an issue to look at mobile support and I'll see what I can do.
https://github.com/eliandoran/breeze-trilium/issues/9

Thanks! :)

**Comment by rodrigoalexandrecosta** [comment]
I also love this theme. I've settled with Breeze Dark and VS Code Dark, for me they're both the best. It's good to see that it's being maintained.
